framework/src/application/network/schema.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { p2pSchemas } from '@liskhq/lisk-p2p';
+
+export const customNodeInfoSchema = {
```

Comment:
Instead of giving full access to the schema itself, isn't it safer to ask user to provide additional schema, and combine them in p2p constructor?
Otherwise, minimally required NodeInfo will be overwritten